Library/SPIClient.m

```diff
@@ -1201,7 +1201,6 @@ - (void)handleGetLastTransactionResponse:(SPIMessage *)m {
             } else {
                 // TH-4X - Unexpected Error when recovering
                 SPILog(@"Unexpected response in get last transaction during - received posRefId:%@ error: %@", [gltResponse getPosRefId], m.error);
```

Comment:
can you perhaps amend the logged message to say that we are ignoring it?
`SPILog(@"Unexpected response in get last transaction during - received posRefId:%@ error: %@. Ignoring.", [gltResponse getPosRefId], m.error);`

Reply:
Added it